apps/server/src/modules/board/service/content-element-update.visitor.ts

```diff
@@ -22,11 +31,12 @@ export class ContentElementUpdateVisitor implements BoardCompositeVisitor {
 		this.throwNotHandled(card);
 	}
 
-	visitTextElement(textElement: TextElement): void {
-		if (this.content instanceof TextContentBody) {
-			textElement.text = this.content.text;
+	visitRichTextElement(richTextElement: RichTextElement): void {
+		if (this.content instanceof RichTextContentBody) {
+			richTextElement.text = sanitizeRichText(this.content.text, this.content.inputFormat);
```

Comment:
aren't we missing a test where the sanitize is called? 
I would immagine in apps/server/src/modules/board/service/content-element-update.visitor.spec.ts 
also, in api test, we could include a test for the rich text format as it is important to find it easy to understand, when someone reads the code, that we have some transformation of the content, and remove all the stuff not allowed